app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java

```diff
@@ -122,7 +128,7 @@ public void navigateToHabitViewFragment() {
         _solo.clickOnView(_solo.getView(R.id.add_habit_fab));
 
         // Create test habit
-        Habit testHabit = new Habit("viewHabitTest", "Test Reason", new Date(), HABIT_ID, EMPTY_HABIT_EVENT_LIST);
+        Habit testHabit = new Habit("viewHabitTest", "Test Reason", new Date(), "MO WE FR", HABIT_ID, EMPTY_HABIT_EVENT_LIST);
```

Comment:
Habits now have a String field of days of the week, which are discussed in DaysTracker.java.

Reply:
Nice! We have the string `MO WE FR` in a lot of places in this file, how about we extract this into a constant?
```
private static final String DAYS_OF_THE_WEEK = "MO WE FR";
```